vmdb/app/views/shared/dialogs/_dialog_field_drop_down_list.haml

```diff
@@ -26,8 +22,14 @@
         $.each(data.values.refreshed_values, function(index, value) {
           var option = '<option ';
           option += 'value="' + value[0] + '" ';
-          if (data.values.checked_value.toString() === value[0].toString()) {
-            option += 'selected="selected" ';
+          if (data.values.checked_value !== null) {
+            if (data.values.checked_value.toString() === value[0].toString()) {
+              option += 'selected="selected" ';
+            }
+          } else {
+            if (index === 0) {
+              option += 'selected="selected" ';
+            }
```

Comment:
lol - thought angular was there to solve this problem :)

`data.values` = hash, with optional key `checked_value`.

condition|code
---|---
no default value|select first element in select box
default value|select default value
bad default value|select nothing